### ISSUE TITLE
communities: several errors when editing settings

### ISSUE THREAD

**Issue by ppanero** [inveniosoftware/invenio-app-rdm#881]
Issues with Community re-naming, steps:
1- I renamed the created community
2- The new name may only contain letters, numbers, dashes ... the original name could contain spaces. 
3- Also, renaming did not affect the header (Title next to the community icon).
4- Also, after saving the new title, the page was unavailable (Renaming affected the URL which is now in line with the new title)

cc @slint @zzacharo

**Comment by fenekku** [comment]
Ah yes I ran into this as well and discovered something. "Renaming" on the community page isn't about renaming! It's about changing the identifier. I recently added this warning: https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/RenameCommunityButton.js#L16 . Either the button label should be ...renamed...  or the functionality should change.
